react/FilterNavigator.js

```diff
@@ -81,6 +81,7 @@ const FilterNavigator = ({
   totalProductsOnMobile = 'hide',
   fullWidthOnMobile = false,
   navigationTypeOnMobile = 'page',
+  priceRangeLayout = 'default',
```

Comment:
```suggestion
  priceRangeLayout = 'slider',
```

I think it would be more clear if the default option had a name representing its actual behavior

Reply:
Agree! It is clearer now.